README.md

```diff
@@ -1,2 +1,113 @@
+
+---
+
+JSON files used in Apostrophe are used by i18next. These resources are automatically converted as pieces through this lodule `@apostrophecms/i18n-static`.
+
+This will add an entry in the admin bar "I18n Static Phrases".
+
+Only admin users have access to this module for now.
+
+<a id="1"></a>
+
+## 1 Installation [&#x2B06;](#contents)
+
+Install the module with `npm i @apostrophecms/i18n-static` then declare it in the modules configuration in `app.js`:
+
+```js
+  modules: {
+    '@apostrophecms/i18n-static': {},
+  }
+```
+
+<a id="2"></a>
+
+
+## 2 Configuration [&#x2B06;](#contents)
+
+The locales configuration comes from the i18n module in a A3 project. Usually, in `modules/@apostrophecms/i18n/index.js`. Example:
+
+```js
+module.exports = {
+  options: {
+    defaultLocale: 'en',
+    locales: {
+      en: { label: 'English' },
+      fr: {
+        label: 'French',
+        prefix: '/fr'
+      },
+      sk: {
+        label: 'Slovak',
+        prefix: '/sk'
+      },
+      es: {
+        label: 'Spanish',
+        prefix: '/es'
+      },
+      'pt-BR': {
+        label: 'Portuguese',
+        prefix: '/pt'
+      },
+      ar: {
+        label: 'Arabic',
+        prefix: '/ar'
+      }
+    }
+  }
+};
+```
+
+I18n-static pieces have a `namespace` field. Namespaces are found in the A3 project. But if resources from specific namespaces should not be converted to i18n-static pieces, they can be excluded by configuring the array `excludeNamespaces` in the module's options.
+
+```js
+  modules: {
+    '@apostrophecms/i18n-static': {
+      options: {
+        excludeNamespaces: [ 'aposEvent', 'aposForm' ]
+      }
+    },
+  }
+```
+
+As the `autopublish` option is set to `true` by default, new pieces will be created as draft and live. Change this option to `false` if `draft` is necessary before publishing them later (the `publish` batch feature will be helpful in this case).
+
+<a id="3"></a>
+
+## 3 Behavior [&#x2B06;](#contents)
+
+Pieces are automatically created based on i18next resources present in the A3 project. They can also be imported as explained in [the following section](#4).
+
+When an i18n-static piece is edited, an ID is generated and stored in the `global` object. If multiple A3 instances are connected to the same database, they have their own ID. When the global one changes, they will detect it and add the missing resource to i18next.
+
+<a id="4"></a>
+
+## 4 Import and export [&#x2B06;](#contents)
+
+The import feature is available in the manager modal: ![](import.jpg)
+
+As the [piece-type-importer plugin](https://github.com/apostrophecms/piece-type-importer) is used, the CSV file used for import must include column names that match exactly the field names (optional empty column names can be excluded).
```

Comment:
I think it's important to talk about how to use the update feature of the module here.